### PR TITLE
Add docs for new PGVector Vector store node

### PR DESCRIPTION
**Pull request by jeanpaul**
Relevant dev-PR:
https://github.com/n8n-io/n8n/pull/10517

### CODE REVIEW

**Review by freakwriter — CHANGES_REQUESTED** [review]
@jeanpaul I pushed a couple commits for formatting stuff so you wouldn't need to think about them. I've added one suggested commit here I wanted you to review--it looked like it was just an oversight in the link name but I wanted to be sure instead of just blindly pushing a commit. Can you review the changes I pushed to make sure they're all still accurate, and see if I interpreted the suggested commit situation properly?

File: docs/advanced-ai/langchain/langchain-n8n.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Learn more about [Agents in LangChain](https://js.langchain.com/docs/modules/age
 Vector stores store embedded data, and perform vector searches on it.
 
 * [In Memory Vector Store](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstoreinmemory/)
+* [Pinecone Vector Store](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector/)
```
[the remark this answers is not in the document]
Comment (jeanpaul) [reply docs/advanced-ai/langchain/langchain-n8n.md:59]: Woops! 🙏

File: docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector.md [hunk 2]
```diff
@@ -23,41 +21,38 @@ You can find authentication information for this node [here](/integrations/built
 
 --8<-- "_snippets/integrations/builtin/cluster-nodes/vector-store-mode.md"
 
-### Parameters for Get Many
+### Get Many parameters
 
-* Table name
-* Prompt: search query.
-* Limit: how many results to retrieve from the vector store. For example, set this to `10` to get the ten best results.
+* **Table name**: Enter the name of the table you want to query.
```
Comment (jeanpaul) [inline docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector.md:28]: These parameters are similar across a few vector stores -- should we apply this change to all of the others too then?

**Comment by jeanpaul** [comment]
> @jeanpaul I pushed a couple commits for formatting stuff so you wouldn't need to think about them.

Other than the comment I just placed regarding syncing the change across other vector stores, I like these changes a lot, thanks!

File: docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector.md (hunk 2 above)
Comment (freakwriter) [reply docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector.md:28]: @jeanpaul if we can, yes! I have some app nodes to update formatting on already, so if you'd like I can just leave myself a note and update those as I'm updating other things. :)

**Review by freakwriter — APPROVED** [review]

File: docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector.md (hunk 2 above)
Comment (jeanpaul) [reply docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector.md:28]: Yes please! That'd be great.

**Pull request merged**